com.spotify.Client.json

```diff
@@ -21,7 +21,8 @@
         "--filesystem=xdg-music:ro",
         "--filesystem=xdg-pictures:ro",
         "--env=LD_LIBRARY_PATH=/app/lib",
-        "--env=TMPDIR=/tmp"
+        "--env=TMPDIR=/tmp",
+        "--env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons"
```

Comment:
this should be done in runtime: https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/commit/3893aaf687788d5af80f0463121c74b536d548b3

dose it not work for you?

Reply:
Honestly I just copy pasted it from what I did for Slack. Will double check, but it wasn't working for Slack at least earlier this year.

Reply:
Yeah appears to work fine now without specifying it manually, thanks for pointing that out! Will update Slack as well.

Reply:
Looks like this is still needed for GNOME (not on KDE though).